content/en/error_tracking/frontend/mobile/reactnative.md

```diff
@@ -5,4 +5,914 @@ code_lang: reactnative
 code_lang_weight: 60
 ---
 
-{{< include-markdown "real_user_monitoring/error_tracking/mobile/reactnative" >}}
\ No newline at end of file
+## Overview 
+
+Enable React Native Crash Reporting and Error Tracking to get comprehensive crash reports and error trends. With this feature, you can access:
+
+-   Aggregated React Native crash dashboards and attributes
+-   Symbolicated React Native (JavaScript and native iOS or Android) crash reports
+-   Trend analysis with React Native Error Tracking
+
+**Note**: In order to symbolicate your stack traces, you must manually upload your source maps and native debug symbols into Datadog.
+
+## Setup
+
+If you have not set up the React Native SDK yet, follow the [in-app setup instructions][16] or see the [React Native setup documentation][17]. Then, follow the steps on this page to enable React Native Crash Reporting and Error Tracking.
+
+ 
+
+### Step 1 - Specify application details in the UI
+
+1. In Datadog, navigate to [**Error Tracking** > **Settings** > **Browser and Mobile** > **Add an Application**][1].
```

Comment:
```suggestion
1. In Datadog, navigate to [**Error Tracking** > **Settings** > **Browser and Mobile**][1] and click **+ New Application**.
```

Reply:
There doesn't seem to be a way to link directly to the screen for adding an application. I've also suggested updating the URL on line `895` to be absolute, since relative links in the docs repo open other doc pages.